internal/interactivity/hiddenCommands.go

```diff
@@ -0,0 +1,10 @@
+package interactivity
+
+var HiddenCommands = []string{
```

Comment:
Using this isntead of the cobra `Hidden` flag because this way they will still show up in `-h` output